aws/policy/storage-services.yaml

```diff
@@ -27,6 +28,7 @@ Statement:
       - s3:PutBucketRequestPayment
       - s3:PutBucketTagging
       - s3:PutBucketVersioning
+      - s3:PutBucketInventoryConfiguration
```

Comment:
This is not an IAM permission that exists. You need to use `PutInventoryConfiguration`.